pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -403,9 +399,6 @@ func (p *PriorityQueue) isPodWorthRequeuing(logger klog.Logger, pInfo *framework
 			queueHint = h
 		}
 	}
-
-	// No queueing hint function is registered for this event
-	// or no queueing hint fn returns the value other than QueueSkip.
```

Comment:
It reads confused to me, I think the logic here is clear, so removed it entirely.